bpdm-common/src/main/kotlin/org/eclipse/tractusx/bpdm/common/dto/response/RegionDto.kt

```diff
@@ -22,15 +22,15 @@ package org.eclipse.tractusx.bpdm.common.dto.response
 import com.neovisionaries.i18n.CountryCode
 import io.swagger.v3.oas.annotations.media.Schema
 
-@Schema(name = "RegionDto", description = "Region within a country")
+@Schema(name = "CountrySubdivisionDto", description = "Country subdivision")
 data class RegionDto(
 
     @get:Schema(description = "Country code")
     val countryCode: CountryCode,
 
-    @get:Schema(description = "Abbreviation or shorthand of the area")
-    val regionCode: String,
+    @get:Schema(description = "The country subdivision code according to ISO 3166-2")
+    val code: String,
 
-    @get:Schema(description = "Describes the full name of the region within a country according to ISO 3166-214")
-    val regionName: String
+    @get:Schema(description = "The name of the country subdivision according to ISO 3166-2")
+    val name: String
```

Comment:
We need to have another DTO since we still need to old Region endpoints to be backwards compatible

Reply:
Fixed it. Now there is a new `CountrySubdivisionDto` besides the old `RegionDto`.